bites_test.py

```diff
@@ -559,6 +567,9 @@ def __init__(self, name, score):
         self.name = name
         self.score = score
```

Comment:
Passing the score here is not ideal. As this test is printing a score of 0 and the real `Player()` class initialises the score to 0, I suggest not passing the score and just having:
```suggestion
        self.score = 0
```